.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: Release
+on: [push, pull_request]
+jobs:
+  main:
```

Comment:
```suggestion
  release-3.4:
```

Reply:
You saved me at least 30 minutes:)

Reply:
Hmm, not sure this worked as intended as there is no longer a `Release` run in checks below. The variable here is only name of the job. It doesn't influence anything so maybe it's worth to revert it to `main` or any other name that is less confusing.

Reply:
I realized that I must merge this PR firstly, otherwise the release test pipeline will always fail, because it always clone relealse-3.4, which doesn't contain this PR yet. 

So let me merge this PR firstly.